selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/CheckoutToRemoteBranchTest.java

```diff
@@ -10,273 +10,143 @@
-    git.waitGitStatusBarWithMess(PULL_MSG);
-    events.clickEventLogBtn();
-    events.waitExpectedMessage(PULL_MSG);
-
-    // Make some change in java file
-    uniqueValue2 = String.valueOf(System.currentTimeMillis());
-    projectExplorer.openItemByVisibleNameInExplorer("GreetingController.java");
-    editor.waitActive();
-    editor.selectLineAndDelete(2);
-    editor.typeTextIntoEditor("//" + uniqueValue);
-    editor.waitTextIntoEditor(uniqueValue);
-    loader.waitOnClosed();
-    editor.waitTabFileWithSavedStatus("GreetingController");
-    loader.waitOnClosed();
-    editor.closeFileByNameWithSaving("GreetingController");
-    editor.waitWhileFileIsClosed("GreetingController");
-
-    // Make some change in jsp file
-    projectExplorer.openItemByVisibleNameInExplorer("index.jsp");
-    loader.waitOnClosed();
-    editor.waitActive();
-    editor.selectLineAndDelete(1);
-    editor.typeTextIntoEditor("<!" + uniqueValue2 + ">");
-    editor.waitTextIntoEditor(uniqueValue2);
-    editor.waitTabFileWithSavedStatus("index.jsp");
-    loader.waitOnClosed();
-    editor.closeFileByNameWithSaving("index.jsp");
-    editor.waitWhileFileIsClosed("index.jsp");
-
-    // Add all files to index and commit
-    projectExplorer.waitAndSelectItem(PROJECT_NAME + "/src/main");
-    menu.runCommand(TestMenuCommandsConstants.Git.GIT, TestMenuCommandsConstants.Git.ADD_TO_INDEX);
-    git.waitGitStatusBarWithMess(TestGitConstants.GIT_ADD_TO_INDEX_SUCCESS);
-    events.clickEventLogBtn();
-    events.waitExpectedMessage(TestGitConstants.GIT_ADD_TO_INDEX_SUCCESS);
-    menu.runCommand(TestMenuCommandsConstants.Git.GIT, TestMenuCommandsConstants.Git.COMMIT);
-    COMMIT_MESS = COMMIT_MESS + new Date().toString();
-    git.waitAndRunCommit(COMMIT_MESS);
-    git.waitGitStatusBarWithMess(TestGitConstants.COMMIT_MESSAGE_SUCCESS);
-    events.clickEventLogBtn();
-    events.waitExpectedMessage(TestGitConstants.COMMIT_MESSAGE_SUCCESS);
-
-    // Make push
-    menu.runCommand(
-        TestMenuCommandsConstants.Git.GIT,
-        TestMenuCommandsConstants.Git.Remotes.REMOTES_TOP,
-        TestMenuCommandsConstants.Git.Remotes.PUSH);
-    git.waitPushFormToOpen();
-    git.selectPushRemoteBranchName(SECOND_BRANCH);
-    git.clickPush();
-    git.waitPushFormToClose();
+    menu.runCommand(GIT, STATUS);
+    git.waitGitStatusBarWithMess(GIT_STATUS_MESS);
 
-    String pushedMessage =
-        "Successfully pushed to git@github.com:"
-            + gitHubUsername
-            + "/springProjectWithSeveralBranches.git";
-    git.waitGitStatusBarWithMess(pushedMessage);
+    performGitPull();
 
-    events.clickEventLogBtn();
-    events.waitExpectedMessage(PUSH_MSG);
+    // change content in the test repo on GitHub
+    changeContentOnGithubSide(pathJavaFile, changeContent, SECOND_BRANCH);
+    changeContentOnGithubSide(pathJspFile, changeContent, SECOND_BRANCH);
 
-    // import from hosted git repository the second project
```

Comment:
Could you, please, explain why did you remove next verifications:
- check presence of previous changes
- check the change in git compare form

Reply:
* Check presence of previous changes is performed (see line 115)
* Check the change in the git compare form is deleted, because it is not a main goal of this selenium test, and it is an unneeded, on my opinion. Also there is the selenium test `GitcompareForm` where is performed needed checks